test/components.js

```diff
@@ -5,7 +5,8 @@ import components from '../src/components'
 
 const blacklist = {
   pre: true,
-  img: true
+  img: true,
+  code: true
```

Comment:
Was this causing errors or something?

Reply:
I think so. I'll verify.

Reply:
It seem we need to initialize the theme provider,